Microsoft.Toolkit.Uwp.SampleApp/SamplePages/FlexPanel/Stepper.xaml

```diff
@@ -0,0 +1,98 @@
+﻿<UserControl
```

Comment:
I don't see a need for this class. Just use two buttons in the sample Page itself

Reply:
Roll your mouse scroll wheel over the control to very quickly increment / decrement.  I was tired of spending so much time clicking that button when trying to do functional testing.